stdlib/pyclbr.pyi

```diff
@@ -1,19 +1,30 @@
 import sys
-from collections.abc import Sequence
+from collections.abc import Mapping, Sequence
 
 __all__ = ["readmodule", "readmodule_ex", "Class", "Function"]
 
-class Class:
+class _Object:
     module: str
     name: str
-    super: list[Class | str] | None
-    methods: dict[str, int]
     file: int
     lineno: int
 
     if sys.version_info >= (3, 10):
         end_lineno: int | None
 
+    parent: _Object | None
+    children: Mapping[str, _Object]
```

Comment:
I'm not totally sure about using Mapping here. Mypy was unhappy with dict because of variance issues, and this does fix that, but normally I'd only reach for Mapping as an argument type

Reply:
Sounds like `Mapping` is the best idea in this case. I think we have other, similar examples in typeshed. Could you add comment to the attribute?

Reply:
Done